spark-market/src/main.sw

```diff
@@ -103,11 +113,21 @@ impl SRC5 for Contract {
     /// * [State] - Represents the state of ownership for this contract.
     #[storage(read)]
     fn owner() -> State {
-        OWNER
+        ownership_owner()
     }
 }
 
 impl SparkMarket for Contract {
+    #[storage(read, write)]
+    fn initialize_ownership(new_owner: Identity) {
+        ownership_initialize_ownership(new_owner);
```

Comment:
can we make this `initialize_ownership()`?

Reply:
it ownership:: function namespace